microsoft-edge/web-platform/ad-selection-api.md

```diff
@@ -0,0 +1,257 @@
+
+To sign up for an Origin Trial token:
+
+1. Go to Microsoft Edge [Origin Trials](https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/) in a new window or tab.
+
+1. In the **Ad Selection API** section, click the **Register** button:
+
+   ![Origin trials](./ad-selection-api-images/origin-trials.png)
+
+1. Complete the form for each domain you intend to use with the Ad Selection API and make sure to provide a valid developer contact:
+
+   ![Registration form](./ad-selection-api-images/reg-form.png)
+
+   * When you receive your OT token, this can be used on the provided domain(s) to enable Ad Selection APIs for testing with supported Microsoft Edge clients.
+
+1. To complete attestation and allow for continued access to the Ad Selection API, you'll need to host a JSON file named `attestations.json` on each domain at a `/.well-known/` path.  For example: `https://contoso.example/.well-known/ad-selection-attestations.json`.
+
+   This JSON file must conform to the following standards below, and must include the created OT token as well as a list of APIs.
+
+
+<!-- ------------------------------ -->
+#### Example attestations.json file
+
+Below is an example of an `attestations.json` JSON file, containing an OT token, which must be published within **30 days** of receiving the initial token:
+
+```json
+{
+    "ad_selection_api_attestations": [
+        {
+            "attestation_parser_version": "2",
+            "attestation_version": "2",
+            "privacy_policy": [
+                "https://contoso.example/privacy/"
+            ],
+            "ownership_token": "<Your Unique Token Here>",
+            "enrollment_site": "https://contoso.example/",
+            "platform_attestations": [
+                {
+                    "platform": "edge",
+                    "attestations": {
+                        "attribution_reporting_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        },
+                        "shared_storage_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        },
+                        "private_aggregation_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        },
+                        "ad_selection_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        }
+                    }
+                },
+                {
+                    "platform": "android",
+                    "attestations": {}
+                }
+            ]
+        }
+    ]
+}
+```
+
+
+<!-- ---------- -->
+###### Important fields and values
+
+* `"platform":` must be `"edge"` or `"android"`.
+
+* `"attestations":` must include `"attribution_reporting_api"`, `"shared_storage_api"`, `"private_aggregation_api"`, and `"ad_selection_api"`.
```

Comment:
tech review "and".  draft Doc has "or" and lacks verb:

![image](https://github.com/user-attachments/assets/72d00413-9e0a-4cef-9443-821b057bbfb3)

Reply:
result:
```
* Valid members of `"attestations":` are `"attribution_reporting_api":`, `"shared_storage_api":`, `"private_aggregation_api":`, and `"ad_selection_api":`.
```